server/lib/analytics/index.js

```diff
@@ -0,0 +1,22 @@
+/**
+ * External dependencies
+ */
+import superagent from 'superagent';
+
+/**
+ * Internal dependencies
+ */
+import config from 'config';
+import { statsdUrl } from '../../../client/lib/analytics/statsd';
```

Comment:
Can we not import from `client/lib/analytics` or `lib/analytics` here? AFAIK webpack will resolve these.